packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
OK please let me know then, because i am afraid i still do not fully understand.